noxfile.py

```diff
@@ -0,0 +1,50 @@
+"""Automation using nox.
+"""
+
+from pathlib import Path
+from typing import Iterator, Tuple
+
+import nox
+
+nox.options.reuse_existing_virtualenvs = True
+
+@nox.session
+def vendoring(session: nox.Session) -> None:
```

Comment:
I'm not keen on involving an extra automation tool (nox) and extra bits of CI logic just for this.

TBH, I think Pradyun's `vendoring` tool is overkill here. I can absolutely see why you'd want it for pip, where there are 25 vendored packages already, but we're only vendoring one, there should be no need to vendor more, and I'd like to make it clear that it's just one. Hopefully even that one will be unnecessary in a few years (if a TOML parser is added to the standard library).

So I think it's easier to update it by hand from time to time than to manage extra code for updating it.